src/plugins/integrations/javascriptPackageJson.ts

```diff
@@ -13,9 +15,27 @@ export const javascriptPackageJson: SiloDiscoveryConfig = {
       optionalDependencies = {},
     } = asJson;
     return [
-      ...Object.keys(dependencies),
-      ...Object.keys(devDependencies),
-      ...Object.keys(optionalDependencies),
+      ...Object.keys(
```

Comment:
So in the plugin run we return the special case map result as the `type`: https://github.com/transcend-io/main/blob/0a75effbef955de06f201fc7e47e3514a369eb4b/integrations/okta/src/pluginsConfiguration.ts#L72

Reply:
So i think instead of just changing the name (that muddles with the context/raw result of what was actually scanned you'll want to change the function signature to actually return the type as well and maybe modify the graphql to accept it as input (maybe not tho i don't remember what graphql has/doesn't have)

https://github.com/transcend-io/cli/blob/0a2924ec103dd300e433d566ca9a1ae7f6e1ebfa/src/plugins/types.ts#L1

Reply:
I should also make this change in the plugin run for these integrations as well?

Reply:
not sure I understand your question.

My comment is about making these cli scan functions a mirror of those pluginConfiguration functions. They should produce the same "raw results" for our classifier. In the pluginConfiguration, they're returning both the original object that was scanned and adding the type as an additional field in the silo disco raw results object. Here the name of the dependencies are being changed so it's not exactly the same.

Reply:
Oh for some reason, github never updates the conversation, unless i refresh, so I missed the last message you sent

Reply:
ya no worries!

Reply:
<img width="264" alt="Screen Shot 2022-08-03 at 3 34 11 PM" src="https://user-images.githubusercontent.com/42789873/182723947-d3c31448-af7e-494b-815f-ab3ce2102e05.png">
This is currently the graphQL input, so I should just add 'type' into this as well?